continuous_integration/travis/install.sh

```diff
@@ -18,6 +18,7 @@ download_code()
     if [ "$PR" != "false" ] ; then
         cd $FULL_SPYDER_CLONE
         git fetch origin pull/$PR/head:travis_pr_$PR
+        git checkout travis_pr_$TRAVIS_PULL_REQUEST
```

Comment:
yes, but if I try to run `pip install -e .[test]` It'll fail because It was in the master branch